unittests/tools/test_codechecker_parser.py

```diff
@@ -0,0 +1,80 @@
+from ..dojo_test_case import DojoTestCase, get_unit_tests_path
+from dojo.tools.codechecker.parser import CodeCheckerParser
+from dojo.models import Test
+
+
+class TestCodeCheckerParser(DojoTestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        testfile = open(
+            get_unit_tests_path() + "/scans/codechecker/cc-report-0-vuln.json"
+        )
+        parser = CodeCheckerParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_finding(self):
+        testfile = open(
+            get_unit_tests_path() + "/scans/codechecker/cc-report-1-vuln.json"
+        )
+        parser = CodeCheckerParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(1, len(findings))
+        finding = findings[0]
+        self.assertEqual("clang-diagnostic-sign-compare", finding.title)
+        self.assertEqual("Medium", finding.severity)
+
```

Comment:
Some checks on file_path and line attributes are missing.

Reply:
thank you, Damien, 
added some extra checks